pcdsdevices/tests/test_sl2k4_slits.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from ophyd.sim import make_fake_device
+
+from pcdsdevices.slits import SL2K4Slits
+
+FakeSL2K4Slits = make_fake_device(SL2K4Slits)
+
+
+@pytest.fixture
+def fake_slits():
+    FakeSlits = make_fake_device(SL2K4Slits)
+    return FakeSlits(prefix='SL2K4:SCATTER', cam='IM6K4:PPM:CAM',
+                     data_source='IMAGE2',
+                     name='sl2k4_slits')
+
+
+def test_slits_initialization(fake_slits):
+    sp = 4.0
+    fake_slits.xwidth.setpoint.put(sp)
+    xwidth_sp = fake_slits.xwidth.setpoint.get()
+    assert pytest.approx(xwidth_sp, 0.1) == sp
```

Comment:
Comment: I'm not sure I understand this test, you're essentially putting to and reading back from a known fake epics signal. None of the code you included is tested here (it's usually not possible to write a meaningful unit test for a device like this unless there are specific coding things added).

I think a more meaningful test would be checking how `cam` and `data_source` are used, but maybe I'm wrong because I don't understand what they are used for.

Lastly, from a test organization perspective, everything from `slits.py` should be tested in `test_slits.py`, we should not make new mini test modules for every new device.